Documentation/Hosting/AmazonGameLift/GameLift.md

```diff
@@ -0,0 +1,105 @@
+# MultiplayerSample Project with Amazon GameLift
+
+This README covers testing and running MultiplayerSample with Amazon GameLift.
+
+## Running with [Amazon GameLift](https://docs.aws.amazon.com/gamelift/index.html)
+
+### Setup
+1. Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), if you don't already have it. You will need it to interact with Amazon GameLift.
+1. Enable the "AWSGameLift" and "MPSGameLift" gem by adding them to MultiplayerSample/Gem/Code/enabled_gems.cmake
+1. Build the server, game launchers, and asset bundler for MultiplayerSample as normal, per [top-level README](/README.md).
+    ```sh
+    cmake --build build\windows --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher AssetBundler --config profile -- /m /nologo
+    ```
+1. Build all the assets
```

Comment:
These lines need to end with two <spaces> to force a line break in markdown.

```suggestion
1. Build the server, game launchers, and asset bundler for MultiplayerSample as normal, per [top-level README](/README.md).  
    ```sh
    cmake --build build\windows --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher AssetBundler --config profile -- /m /nologo
    ```
1. Build all the assets.  
```

Examining the markdown preview shows that all the steps are on a single line

Reply:
Did a markdown pass and found lots of errors. Fixed! Thanks!